### ISSUE TITLE
Number of common classes in Synthia dataset

### ISSUE THREAD

**Issue by wangkaihong** [liyunsheng13/BDL#39]
Hi Yunsheng,

I noticed that in your paper you mentioned:

>  For SYNTHIA [28], we use the SYNTHIA-RAND-CITYSCAPES
set which contains 9, 400 images with the resolution 1280×
760 and 16 common categories with Cityscapes [5].

which means only 16 classes should be involved in the training and the evaluation in the experiment, but I found in the configuration of the dataset of Synthia that:

`        self.id_to_trainid = {3: 0, 4: 1, 2: 2, 21: 3, 5: 4, 7: 5,
                              15: 6, 9: 7, 6: 8, 16: 9, 1: 10, 10: 11, 17: 12,
                              8: 13, 18: 14, 19: 15, 20: 16, 12: 17, 11: 18}
` 

so there are 19 classes in total, just like GTA V and cityscapes. Is that an inconsistency between the code and the paper or did I miss anything?

Thanks,
Kaihong

**Comment by liyunsheng13** [comment]
No, there is not any inconsistency between the code and paper. During evaluation, I only evaluate 16 of the common classes.

**Comment by wangkaihong** [comment]
> No, there is not any inconsistency between the code and paper. During evaluation, I only evaluate 16 of the common classes.

That sounds fair. But what about the training process? Did you train the model with 19 classes?

I noticed that your dataset code might be borrowed from https://github.com/wasidennis/AdaptSegNet, which provides only the GTA V -> cityscapes experiment script. Is that the reason you are using this configuration?

Thanks again,
Kaihong
